.prettierignore

```diff
@@ -18,3 +18,6 @@ bin/yarn*
 **/*.generated.ts
 /tools/dist
 /patches
+
+# all contributors bot configuration file
+.all-contributorsrc.json
```

Comment:
Is this required?

Reply:
I'm not sure if the all-contributors bot formats the JSON the way Prettier wants it, so that's why I'm listing it here.
We could drop these lines, and see what happens?

Reply:
Leave it for now, we can check later if prettier would change it.